numpy/core/_type_aliases.py

```diff
@@ -107,14 +107,16 @@ def _add_aliases():
         if name in ('longdouble', 'clongdouble') and myname in allTypes:
             continue
 
-        allTypes[myname] = info.type
-
-        # add mapping for both the bit name and the numarray name
-        sctypeDict[myname] = info.type
+        # Add to the main namespace if desired:
```

Comment:
Not sure what this comment means. `allTypes` is not in the main namespace, and I think we'd like to keep it that way. Remove?

Reply:
allTypes is what ends up in the main namespace:
```
# Now add the types we've determined to this module
for key in allTypes:
    globals()[key] = allTypes[key]
    __all__.append(key)
```

Reply:
Ah okay, missed that bit.